tests/index/test_index_document_meilisearch.py

```diff
@@ -25,6 +25,15 @@ def test_add_documents(empty_index, small_movies):
     assert index.get_primary_key() == "id"
     assert update.status == "succeeded"
 
+def test_add_documents_empty(empty_index):
+    """Tests adding empty string as documents to a clean index."""
+    index = empty_index()
+    with pytest.raises(Exception) as e_info:
+        response = index.add_documents('')
+    assert e_info.value.message == "A json payload is missing."
```

Comment:
```suggestion
```
We don't test the error message, as they may be changed, which will require a lot of test maintenance.